### PR TITLE
Page for request form view only

### PR DESCRIPTION
**Pull request by ShellyXueHan**
To add a page that displays the content of a request:
1. get request info from github
2. normaliza data
3. display request in a form, same as form used in creating new request.
4. for reviewers, provide option to approve/reject (actual actions not implemented yet)
5. a bit style updates to fit the form
6. used semantic-ui-css instead

Devops update: (should be in another pr, but then this one feature got too distracted and forgot to managed that....)
1. setup the jenkins job
2. a bit update in pipeline templates

### CODE REVIEW

File: api/src/libs/gh-utils/gh-helpers.js [hunk 1]
```diff
@@ -46,3 +47,40 @@ export const jsonReader = (jsonData, paths = null) => {
   }
   throw Error(`Cannot process data for the path: ${paths}.`);
 };
+
+/**
+ * Validate and filter PR:
```
Comment (patricksimonian) [inline api/src/libs/gh-utils/gh-helpers.js]: maybe a better description of what the fn does? Not really sure what you mean by filtering a pr

File: web/__tests__/ProtectedRoute.test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  it('render content when user has correct role', () => {
+    const { container } = render(
+      <BrowserRouter>
+        <ProtectedRoute component={TestComponent} authCode={ACCESS_CONTROL.REQUESTER_ROLE} />
+      </BrowserRouter>
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  // TODO: Issue with testing on redirec: Maximum update depth exceeded.
```
Comment (patricksimonian) [inline web/__tests__/ProtectedRoute.test.js]: maybe mock out redirect to just to return a react component you can test for?

File: web/src/components/Request/RequestList.js [hunk 3]
```diff
@@ -19,11 +23,15 @@ export const RequestList = ({ requests }) => {
         key={request.number}
         onClick={() => {
           onClick(request.number);
+          // onClick(request.prContent.id);
```
Comment (patricksimonian) [inline web/src/components/Request/RequestList.js]: commented out code

File: web/__tests__/ProtectedRoute.test.js (hunk 2 above)
Comment (ShellyXueHan) [reply web/__tests__/ProtectedRoute.test.js]: mocking not working, will try later

**Pull request merged**